packages/forms/backend/sirius-components-collaborative-forms/src/main/resources/schema/form.graphqls

```diff
@@ -319,6 +328,7 @@ extend type Mutation {
   editSelect(input: EditSelectInput!): EditSelectPayload!
   editTextfield(input: EditTextfieldInput!): EditTextfieldPayload!
   updateWidgetFocus(input: UpdateWidgetFocusInput!): UpdateWidgetFocusPayload!
+  selectListItem(input: SelectListItemInput!): SelectListItemPayload!
```

Comment:
The changelog and the commit message are not representing the changes made in this commit. It's about adding the list widget to the form description editor and adding additional capabilities to this existing widget. The later, at least, being an API breakage (existing list description would now crash with a NPE) should be documented as such.

I'm not sure that `selectListItem` really represents the action of clicking on a list item...